nocgo.go

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build !cgo && darwin
+// +build !cgo,darwin
```

Comment:
Please add a suffix '_darwin.go' to this file instead of specifying darwin here.